tap_freshdesk/__init__.py

```diff
@@ -152,6 +152,7 @@ def sync_tickets_by_filter(bookmark_property, predefined_filter=None):
     if predefined_filter:
         params['filter'] = predefined_filter
 
+    query_time = singer.utils.now()
```

Comment:
no tests makes it hard to believe it works in all cases

Reply:
I did run this end to end but yeah, looks like a lot of the singer taps don't have tests. Maybe that's something worthwhile to contribute to, although you'd still only be able to test the transformation of the data in the tap.